src/css/controls/imports/slider.less

```diff
@@ -118,6 +118,10 @@
     z-index: 1;
     outline: none;
 
+    &&:not(.jw-no-focus)&:focus .jw-slider-container .jw-rail {
```

Comment:
outputs `.jw-slider-time.jw-slider-time:not(.jw-no-focus).jw-slider-time:focus .jw-slider-container .jw-rail` - should it be shortened to `.jw-slider-time:focus:not(.jw-no-focus) .jw-slider-container .jw-rail` ?